### PR TITLE
Requires condition based on the presence of files on the file system

### PR DESCRIPTION
**Pull request by alvarosanchez**

### CODE REVIEW

File: inject/build.gradle [hunk 1]
```diff
@@ -41,6 +41,7 @@ dependencies {
     compileOnly dependencyVersion("validation")
 
     testCompile dependencyVersion("validation")
+    testCompile project(":inject-groovy")
```
Comment (jameskleeh) [inline inject/build.gradle]: Is this change intentional?

**Comment by graemerocher** [comment]
I think this PR should be changed to use `resource=".."` and instead use the `ResourceLoader` concept. That what you would express `@Requires(resource="file:/foo/bar.txt")` but could also do  `@Requires(resource="classpath:foo/bar.txt")`

File: inject/build.gradle (hunk 1 above)
Comment (alvarosanchez) [reply inject/build.gradle]: The test wasn't working without it

**Comment by alvarosanchez** [comment]
> I think this PR should be changed to use `resources=".."` and instead use the `ResourceLoader` concept. That what you would express `@Requires(resources="file:/foo/bar.txt")` but could also do `@Requires(resources="classpath:foo/bar.txt")`

That's fine. I'll work on it.
